packages/fx-core/tests/plugins/resource/localdebug/unit/index.test.ts

```diff
@@ -64,14 +64,14 @@ describe(LocalDebugPluginInfo.pluginName, () => {
         numConfigurations: 5,
         numCompounds: 2,
         numTasks: 9,
-        numLocalEnvs: isMultiEnvEnabled() ? 29 : 30,
+        numLocalEnvs: isMultiEnvEnabled() ? 30 : 30,
```

Comment:
Since the env count is consistent whether the `MultiEnv` feature flag is enabled or not, we can just set to a fix number.

Reply:
Not all (just for tab+auth it's the same), and it's possible to adjust when adding new features to multi-env.

Reply:
OK, That would make sense.